pyright/master/exclude.txt

```diff
@@ -8,6 +8,7 @@ examples/project_fully_featured
 examples/assets_dbt_python
 examples/assets_modern_data_stack
 examples/assets_smoke_test
+examples/docs_projects
```

Comment:
I'm not sure exactly what `docs_projects` is used for but seems questionable to exclude the entire directory from type-checking